splunk-otel-android-volley/README.md

```diff
@@ -0,0 +1,59 @@
+
+# Splunk OpenTelemetry Android Volley Instrumentation
+
+> :construction: &nbsp;Status: Experimental
+
+This directory contains the Splunk instrumentation for the [Volley](https://google.github.io/volley/)
+HTTP client library. If you use the Volley HTTP client in your Android application, you can
+leverage this instrumentation to help simplify tracing and capture of HTTP headers.
+
+## Add to your project
+
+In addition to the [main Splunk Android SDK dependency](https://github.com/signalfx/splunk-otel-android#getting-the-library),
+you will also need to add the `splunk-otel-android-volley` dependency to your `build.gradle.kts`:
+
+
+```gradle
+dependencies {
+    ...
+    implementation("com.splunk:splunk-otel-android-volley:0.16.0")
+    ...
+}
+```
+
+## How to use VolleyTracing
+
+See the class `VolleyExample.java` file in the sample application in this repository for an
+example of how to create an instrumented client HTTP call.
+
+First, you will create an instance of `VolleyTracing` by passing your `splunkRum` instance
+to the builder:
+
+```java
+VolleyTracing volleyTracing = VolleyTracing.builder(splunkRum).build();
+```
+You can also add extra request/response headers to capture to the builder.
+These headers will be attached to the RUM Volley HTTP client spans with
+a`http.response.header.` prefix.
+
+Next, get an instance of `HurlStack` from your `volleyTracing` instance:
+
+```java
+HurlStack hurlStack = volleyTracing.newHurlStack();
+```
+
+Then, use the `hurlStack` to create your request queue,
+
+```java
+RequestQueue queue = new RequestQueue(new NoCache(), new BasicNetwork(hurlStack), 1,
+                        new ExecutorDelivery(Executors.newSingleThreadExecutor()));
+queue.start();
+```
+
+To make instrumented Volley HTTP requests, just use the `RequestQueue` like you normally would!
```

Comment:
I suspect that you should not be using an exclamation mark here. I would have stopped at `hurlStack`, we can assume that the users know how to use this library and can figure out the rest on their own.

Reply:
Ok, I'll just drop that last bit and shorten it. Thanks.